scripts/environment.sh

```diff
@@ -86,6 +89,13 @@ do
         HOSTNAME="${1#*=}"
         shift 1
         ;;
+        --restrict*)
+        RESTRICTED=true
+        if [ "${1#*=}" != "--restrict" ] ; then
+          RESTRICTED_PASSWORD="${1#*=}"
+        fi
```

Comment:
I don't completely get this part...

the `restrict` should come with required parameter I think, and you should check `--restrict=*` as at the other options.

I don't see any yield of defining default value for the restricted mode when it is optional anyway. Moreover, there is no documentation about this default password which would be necessary.

Reply:
> I don't see any yield of defining default value for the restricted mode when it is optional anyway. Moreover, there is no documentation about this default password which would be necessary.

Ehm, its here: https://github.com/ethersphere/bee-factory/pull/69/files#diff-f3736fd0d9f6c379e7622c6230d742bb019c1120223e1c3857bb1808d27d5413R11

Reply:
I meant in the README...